templates/_content_download_download.ftl

```diff
@@ -199,20 +199,20 @@
                                     <ul>
```

Comment:
How this file and [this other file](https://github.com/kiegroup/drools-website/blob/main/download/download.adoc) correlate? It looks like a duplicate 🤔

Reply:
one FTL is a macro, the other is the actual content for download.
The problem with the previous PR was updating only the previous awestruct (which, once we confident this migration to jbake is indeed stable, we can elide from repo)

Reply:
Ok thanks, yes I think we can remove them